### PR TITLE
Enable CLI to support creating multiple vm instances

### PR DESCRIPTION
**Pull request by irwen**
- Enable the cli to create multiple vm instances in one request.
- Add json support for `vm ready` cmd.
- Add json support for `vm detail` command.

### CODE REVIEW

**Comment by kramimus** [comment]
so just packing everything in a big array works?

one thing to double check, if you order like 4 vms at once, do they appear on the same subnet?

File: SoftLayer/CLI/virt/create.py [hunk 1]
```diff
@@ -266,15 +275,7 @@ def cli(env, **args):
...
-        result = vsi.create_instance(**data)
-
-        table = formatting.KeyValueTable(['name', 'value'])
-        table.align['name'] = 'r'
-        table.align['value'] = 'l'
-        table.add_row(['id', result['id']])
-        table.add_row(['created', result['createDate']])
-        table.add_row(['guid', result['globalIdentifier']])
-        output.append(table)
+        result = vsi.create_instances(config_list)
```
Comment (kramimus) [inline SoftLayer/CLI/virt/create.py]: oh nice, I see now, they just have a plural version too, lol

**Comment by kramimus** [comment]
so it looks like you will need to handle things slightly differently, at least for "slcli nas grant-access" since you don't have an order id and just vm ids, but this looks good so far

**Comment by irwen** [comment]
Yea I'll try to test if grant-access will work with vmId first. Need to figure something out if it doesn't.

**Comment by irwen** [comment]
Still focusing on get the VM cluster launched first.

**Pull request merged**